core/templates/dev/head/pages/topic_editor/topic_editor_navbar_directive.html

```diff
@@ -74,14 +74,20 @@
 
   <li ng-class="{'active': getTabStatuses().active === 'subtopics'}" class="icon">
     <a uib-tooltip="Subtopics Editor" tooltip-placement="bottom" ng-click="selectSubtopicsTab()">
-      <i class="fa fa-book material-icons"></i>
+      <i class="material-icons">library_books</i>
```

Comment:
This is prescient :) Out of curiosity, what caused you to get rid of fa?

/cc @hoangviet1993 since the two of us were just talking about doing exactly this sort of thing earlier today :)

Reply:
No major reason, just thought I'd have uniformity across navbar icons since I think material icons are used in most places.
Also, they looked better :).